bundles/org.eclipse.passage.lic.jface/src/org/eclipse/passage/lic/internal/jface/dialogs/licensing/ImportLicenseDialog.java

```diff
@@ -70,6 +70,12 @@ protected void updateButtonsEnablement() {
 
 	}
 
+	@Override
+	protected String defaultMessage() {
+		// TODO Auto-generated method stub
+		return null;
```

Comment:
empty string?

Reply:
One of the following PRs will be dedictaed to further implementation on the dialog.